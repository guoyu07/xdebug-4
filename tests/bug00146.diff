003-                            >=> array ('te"st\'s' => 42)