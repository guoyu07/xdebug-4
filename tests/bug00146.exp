TRACE START [%d-%d-%d %d:%d:%d]
%w%f %w%d     -> foo(array ('te"st\'s' => 42)) /%s/bug00146.php:10
                           >=> array ('te"st\'s' => 42)
%w%f %w%d     -> file_get_contents('/tmp/%s') /%s/bug00146.php:12