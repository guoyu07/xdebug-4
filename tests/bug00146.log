
---- EXPECTED OUTPUT
TRACE START [%d-%d-%d %d:%d:%d]
%w%f %w%d     -> foo(array ('te"st\'s' => 42)) /%s/bug00146.php:10
                           >=> array ('te"st\'s' => 42)
%w%f %w%d     -> file_get_contents('/tmp/%s') /%s/bug00146.php:12
---- ACTUAL OUTPUT
TRACE START [2010-05-18 17:18:09]
    0.0009     622344     -> foo(array ('te"st\'s' => 42)) /Users/dsp/dev/c/xdebug/tests/bug00146.php:10
    0.0010     622416     -> file_get_contents('/tmp/xdt4bf2cbd10dd372.21621524.xt') /Users/dsp/dev/c/xdebug/tests/bug00146.php:12
---- FAILED
