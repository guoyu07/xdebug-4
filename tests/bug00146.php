<?php
	$tf = xdebug_start_trace('/tmp/'. uniqid('xdt', TRUE));

	function foo($a)
	{
		return $a;
	}

	$array = array("te\"st's" => 42);
	foo($array);

	echo file_get_contents($tf);
	unlink($tf);
?>
