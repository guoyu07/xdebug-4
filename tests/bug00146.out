TRACE START [2010-04-16 12:23:43]
    0.0007     626424     -> foo(array ('te"st\'s' => 42)) /Users/johann/src/xdebug/tests/bug00146.php:10
    0.0008     626472     -> file_get_contents('/tmp/xdt4bc856cf7e16f3.03623443.xt') /Users/johann/src/xdebug/tests/bug00146.php:12